collections-csharp/IEnumeralbleVsICollectionVsIListVsList/Tests/Test.cs

```diff
@@ -0,0 +1,63 @@
+using IEnumeralbleVsICollectionVsIListVsList;
+using NUnit.Framework;
+using System;
+
+namespace Tests
+{
+    public class Tests
```

Comment:
Both The **file** and the class that contains the type should end with *UnitTest so I suggest to change this file to CollectionInterfacesUnitTest os something similar.